game_python/LoginLogic.py

```diff
@@ -47,7 +47,7 @@ def send_credential(self):
         data is incorrect.
         """
         if len(self.username) > 0 and len(self.room_number) == 4:
-            print(f'user: {self.username}, room: {self.room_number}')
+            print(f"user: {self.username}, room: {self.room_number}")
```

Comment:
ale tu to aż jestem ciekawy co ci zrobiły te ' 
[String Quotes](https://peps.python.org/pep-0008/#string-quotes)
In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in